net/unison/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        bcpierce00 unison 2.51.4 v
+github.setup        bcpierce00 unison 2.51.5 v
```

Comment:
I'd suggest we update Unison itself in a distinct step. In particular, the rest of this may take a bit of time to shake out, but Unison could be updated right now with little difficulty.

Reply:
Do you mind if we also add `arm64` as supported architecture for unison (like ocaml does)?

Reply:
That's fine, but again, one can do this in two pull requests and thus get Unison updated immediately.

Reply:
Done in https://github.com/macports/macports-ports/pull/13672. I still need to remove unison update from this PR.

Reply:
I've merged that PR.